cli/ops/tls.rs

```diff
@@ -167,6 +171,21 @@ fn load_keys(path: &str) -> Result<Vec<PrivateKey>, ErrBox> {
   Ok(keys)
 }
 
+#[allow(dead_code)]
+pub struct TlsListenerResource {
+  tcp_listener_rid: ResourceId,
```

Comment:
I thought this was clever but there's no way to close "child" resource from within parent resource 😢